packages/core/src/controllers/TransactionsController.ts

```diff
@@ -34,9 +34,9 @@ export const TransactionsController = {
     return sub(state, () => callback(state))
   },
 
-  async fetchTransactions() {
+  async fetchTransactions(address?: string) {
```

Comment:
@0xAsimetriq 

What would be the best approach connecting to Account state inside the `TransactionsController`? Or do you think we should do it?

Instaed passing the new address on the time we changed the wallet, I'd love to watch the account state and have the updated state here.

Reply:
You shouldn't connect them, instead subscribe to address change in transaction list view. When address changes re-triger tx fetching logic from within that page by calling appropriate method(s) of `TransactionsController`